src/sayt/results.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * The type of an autocomplete result.
```

Comment:
Not sure if it's possible, but could we link to line 17 of sayt/autocomplete.ts ?

Reply:
They are kind of already linked, if someone clicks through from the definition on line 17 of sayt/autocomplete.ts they will end up at this definition.